ggml/src/ggml-cpu/ggml-cpu.c

```diff
@@ -7662,8 +7632,8 @@ static void ggml_compute_forward_mul_mat_id(
     ggml_vec_dot_t    const vec_dot         = type_traits_cpu[type].vec_dot;
     enum ggml_type    const vec_dot_type    = type_traits_cpu[type].vec_dot_type;
     ggml_from_float_t const from_float      = type_traits_cpu[vec_dot_type].from_float;
-    int64_t           const matmul_num_cols = type_traits_cpu[type].ncols;
-    ggml_gemv_t       const gemv            = type_traits_cpu[type].gemv;
+    //int64_t           const matmul_num_cols = type_traits_cpu[type].ncols;
+    //ggml_gemv_t       const gemv            = type_traits_cpu[type].gemv;
```

Comment:
Look for me it have not be write for dynamic repack but only with "native" Q4_0_N_M packing.
leave it commented it need some work to be usable on dynamic repacking.

Reply:
We should try to fix to keep support for aarch64 types with models with experts.